CMake/SlicerMacroBuildModuleQtLibrary.cmake

```diff
@@ -210,6 +210,7 @@ macro(SlicerMacroBuildModuleQtLibrary)
     file(GLOB headers "${CMAKE_CURRENT_SOURCE_DIR}/*.h")
     install(FILES
       ${headers}
+      ${MODULEQTLIBRARY_UI_CXX}
```

Comment:
```suggestion
      ${MODULEQTLIBRARY_UI_CXX} # Correspond to the "ui_*.h" files
```

Reply:
In light of the discussion below, I don't think this is needed anymore.